go.mod

```diff
@@ -30,9 +30,11 @@ require (
 	github.com/sourcegraph/go-ctags v0.0.0-20230111110657-c27675da7f71
 	github.com/sourcegraph/log v0.0.0-20230203201409-49ac5ad3f2ce
 	github.com/sourcegraph/mountinfo v0.0.0-20230106004439-7026e28cef67
+	github.com/sourcegraph/sourcegraph/protos/frontend/indexedsearch v0.0.0-20230304041038-7df663cf89c4
```

Comment:
IIRC, we talked about defining the protos in the `zoekt` repository so that we didn't need to add a dependency on `sourcegraph/sourcegraph` to the `zoekt` repo. This way we don't add circular dependency between zoekt and the sourcegraph monorepo. Was there a reason you decided against this?

Reply:
There are two client server interactions involving Zoekt here:

Sourcegraph's frontend configuration point (server) <-> zoekt indexserver (client): (this PR)to fetch information related to what repositories to index

zoekt-webserver (server) <-> sourcegraph's search backend (client): to fetch search results

---

My understanding was that we'd have each server implementation be responsible for defining their own API (and, thus, exporting their own protobuf definitions). This would mean that the frontend configuration protos would live in a separate package in the sourcegarph monorepo (to avoid needing to import all of sourcegraph), and the zoekt-webserver API protos would live in the Zoekt repo. This shouldn't cause a circular dependency issue.

However, I am happy to move all of the protobuf definitions entirely to the Zoekt repository if you prefer that workflow.

Reply:
I expect we'll have an easier time maintaining this all by moving the definitions to the Zoekt repository. This way, Zoekt defines the server it needs to be implemented, we don't have a circular import dependency, and the protos are versioned along with Zoekt, so we can version bump in one place (update Zoekt) rather than two (update sourcegraph/sourcegraph and zoekt)

Reply:
Also, it feels more in the spirit of the standalone project to avoid importing from `sourcegraph/sourcegraph`

Reply:
I'd be curious to hear @keegancsmith and @stefanhengl's takes on this as well.

Reply:
The definitions are now defined within the Zoekt project.